website/docs/dbt-cli/cli-overview.md

```diff
@@ -3,7 +3,7 @@ title: "CLI overview"
 description: "Run your dbt project from the command line."
 ---
 
-dbt Core ships with a command-line interface (CLI) for running your dbt project. The dbt CLI is free to use and available as an [open source project](https://github.com/dbt-labs/dbt-core).
+dbt Core ships with a command-line interface (CLI) for running your dbt project. dbt Core and it's CLI are free to use and available as an [open source project](https://github.com/dbt-labs/dbt-core).
```

Comment:
is this page referenced anywhere? if not, we should prob remove it because its one of those pages that's  not even in the sidebar or anything https://docs.getdbt.com/dbt-cli/cli-overview
![Screenshot 2023-10-27 at 16 44 25](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/bcd5ce59-b1ff-49a9-9d21-73667698f670)

Reply:
It still gets hundreds of views a week so let me check with PM about whether we should remove it entirely or move this content elsewhere. I'll create a follow up PR based on their answer

Reply:
I agree we should remove it (and move content if we need to.) Maybe redirect it to https://docs.getdbt.com/docs/core/about-dbt-core ?

Also check if any docs link to it!